odh-dashboard/base/odh-dashboard-crd.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: odhdashboards.opendatahub.io
+spec:
+  group: opendatahub.io
+  scope: Namespaced
+  names:
+    plural: odhdashboards
+    singular: odhdashboard
```

Comment:
this name makes me think you're creating an OdhDashboard and it's possible to create multiple dashboards.  I think we might want to call it an OdhDashboardConfig.

Reply:
Changed the name, I will also change the expected value in https://github.com/opendatahub-io/odh-dashboard/pull/238